rogue-thi-app/pages/personal.jsx

```diff
@@ -340,11 +351,24 @@ export default function Personal () {
   )
 }
 
+async function getCommitHash () {
+  return new Promise((resolve, reject) => {
+    exec('git rev-parse HEAD', (err, stdout) => {
```

Comment:
I think this should be embedded in the Dockerfile rather than in js. I am not even sure wether we can rely on `git` being available when building with docker